families/identity/sawtooth_identity/Cargo.toml

```diff
@@ -0,0 +1,59 @@
+#
+#  Copyright 2018 Bitwise IO, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------------
+
+[package]
+name = "sawtooth_identity"
+version = "0.1.0"
+authors = ["Bitwise IO"]
+build = "build.rs"
+
+[dependencies]
+clap = "2"
+protobuf = "2.0"
+cfg-if = "0.1"
+
+[target.'cfg(target_arch = "wasm32")'.dependencies]
+rust_crypto = {git = "https://github.com/agunde406/rust-crypto", branch="wasm_sha2"}
+sabre-sdk = {git = "https://github.com/hyperledger/sawtooth-sabre"}
+
+[target.'cfg(unix)'.dependencies]
+rust-crypto = "0.2"
+sawtooth_sdk = { path = "../../../sdk/rust" }
```

Comment:
We should pull this in from crates now: https://crates.io/crates/sawtooth-sdk

Reply:
It seems like we should be testing this code against the codebase that it's sitting in, instead of what's been published, as it might otherwise get out of sync with what's in master.

Reply:
I agree with @knkski. If it is in the core repo, it should be tested against the current code, and not a release version of the code. It will be easier to see inconsistencies.